### PR TITLE
Thu/user info improvements

### PR DESCRIPTION
**Pull request by csm-thu**
Improvements of the UserInfo components:

- add placeholder for a documentation PDF file downloadable from the UserInfo menu
- add language settings in UserInfo menu
- use redux for authentication process

Notes:

- jest tests have been disabled until we find a way to make them work with the new "ConnectedApp" component (connected to redux store)
- waiting for the PR for the UserInfo component to be merged : https://github.com/Cosmo-Tech/webapp-component-ui/pull/2

### CODE REVIEW

File: src/layouts/TabLayout/TabLayout.js [hunk 1]
```diff
@@ -76,6 +77,10 @@ const TabLayout = props => {
   const classes = useStyles();
   const { t } = useTranslation();
   const location = useLocation();
+  const languages = {
+    fr: 'Français',
```
Comment (COSMO-VMI) [inline src/layouts/TabLayout/TabLayout.js]: F id after E in alphabetical order, and to be consistent with Sign In view, I suggest to list in this same order.

**Review by COSMO-VMI — APPROVED** [review]
LGTM, GG

File: public/locales/en/translation.json [hunk 2]
```diff
@@ -154,5 +154,14 @@
         }
       }
     }
+  },
+  "genericcomponent": {
+    "userinfo": {
+      "button": {
+        "changeLanguage": "Change language",
+        "downloadDocumentation": "Download documentation",
+        "logout": "Log out"
```
Comment (COSMO-VMI) [inline public/locales/en/translation.json]: 'Sign out' seems more appropriate, for 'Sign in' is used in the Sign In view.

**Review by COSMO-VMI — APPROVED** [review]
LGTM, GG

File: src/state/reducers/auth/AuthReducer.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  status: AUTH_STATUS.ANONYMOUS
+};
+
+export const authReducer = createReducer(authInitialState, (builder) => {
+  builder
+    .addCase(AUTH_ACTIONS_KEY.REQUEST_LOG_IN, (state, action) => {
+      state.status = AUTH_STATUS.CONNECTING;
+    })
+    .addCase(AUTH_ACTIONS_KEY.REQUEST_LOG_OUT, (state, action) => {
+      Auth.signOut();
```
Comment (jreynard-code) [inline src/state/reducers/auth/AuthReducer.js]: Should not there, should be done in a redux-saga function.
Reducers are pure function (see [reducer](https://redux.js.org/tutorials/fundamentals/part-3-state-actions-reducers#rules-of-reducers))

**Review by jreynard-code — APPROVED** [review]
LGTM! 
GG ;)

**Pull request merged**